rotkehlchen/globaldb/cache.py

```diff
@@ -107,8 +127,22 @@ def globaldb_delete_general_cache(
 ) -> None:
     """Deletes from cache in globaldb. If value is None deletes all entries with the provided
     cache key. Otherwise, deletes only single entry `key - value`."""
-    cache_key = _compute_cache_key(key_parts)
+    cache_key = compute_cache_key(key_parts)
     if value is None:
         write_cursor.execute('DELETE FROM general_cache WHERE key=?', (cache_key,))
     else:
         write_cursor.execute('DELETE FROM general_cache WHERE key=? AND value=?', (cache_key, value))  # noqa: E501
+
+
+def globaldb_delete_general_cache_like(
+        write_cursor: DBCursor,
+        key_parts: Iterable[Union[str, GeneralCacheType]],
+) -> None:
+    """Function to delete globaldb cache. Deletes all the values that start with key."""
+    assert all(
```

Comment:
same here. Why did you think you need to do this? If we were doing this all over the code then each function should be ful of assertions at the start